.github/workflows/CI.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     - name: Checkout MIWG Test Suite
       run: |
         mkdir -p ${{ env.MIWG_PATH }}
-        git clone https://github.com/bpmn-io/bpmn-miwg-test-suite.git ${{ env.MIWG_PATH }}
+        git clone https://github.com/bpmn-miwg/bpmn-miwg-test-suite.git ${{ env.MIWG_PATH }}
```

Comment:
Are we sure we want to do this? 

The reason we point to a fork here is that we are able to control at which point new diagrams are pulled in (cf. renamings in their test suite).

I'd love to keep our fork, and simply update it to a good known last version.

Reply:
I can do that, but then we still need to remember to update the fork from time to time.

Reply:
I updated [the fork](https://github.com/bpmn-io/bpmn-miwg-test-suite) to the latest commit. Also, I disabled GH Actions for the fork, so that we don't execute any workflows with future updates.

So, for now we can use the current test suite from the fork until the next test cases are added in 2023.

Reply:
Does dependabot support fork updating? If that is the case we could configure it to notify us (via a PR) when new changes in upstream get added.

Reply:
We could setup a cron job to check this.

Reply:
https://github.com/bpmn-io/bpmn-miwg-test-suite/issues/2